src/ts/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,232 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { PureComponent } from 'react';
+import { ComponentProps } from '../../types';
+import Button from '../forms/button';
+import SpacedGroup from '../misc/spaced-group';
+
+export interface PaginationProps extends ComponentProps {
+  /**
+   * className
+   */
+  className?: string;
+  /**
+   * is disabled
+   * @default false
+   */
+  disabled?: boolean;
+  /**
+   * items count per page
+   */
+  pageSize: number;
+  /**
+   * current page number
+   */
+  currentPageNumber: number;
```

Comment:
Is this zero indexed? Could you add to the doc comment whether zero or one.

Reply:
Updated as it starts from 1